pkg/cli/admin/upgrade/channel/channel.go

```diff
@@ -86,31 +91,38 @@ func (o *Options) Run() error {
 
 	if o.Channel == cv.Spec.Channel {
 		if cv.Spec.Channel == "" {
-			fmt.Fprint(o.Out, "info: Cluster channel is already clear\n")
+			fmt.Fprint(o.Out, "info: Cluster channel is already clear (no change)\n")
 		} else {
-			fmt.Fprintf(o.Out, "info: Cluster is already in %s\n", cv.Spec.Channel)
+			fmt.Fprintf(o.Out, "info: Cluster is already in %s (no change)\n", cv.Spec.Channel)
 		}
 		return nil
 	}
 
 	if len(cv.Status.Desired.Channels) > 0 {
-		found := false
+		found, known := false, false
 		for _, channel := range cv.Status.Desired.Channels {
 			if channel == o.Channel {
 				found = true
+			}
+			if channel == cv.Spec.Channel {
+				known = true
+			}
+			if found && known {
 				break
 			}
 		}
-		if !found {
+		if o.Channel == "" {
+			if known && !o.AllowExplicitChannel {
+				return fmt.Errorf("You are requesting to clear the update channel. The current channel %q is one of the available channels (%s), you must pass --allow-explicit-channel to continue\n", cv.Spec.Channel, strings.Join(cv.Status.Desired.Channels, ", "))
```

Comment:
I think we can drop the ` (%s)` here.  Folks interested in clearing a channel probably didn't intend to type a viable channel, so they don't need a set of hints for their next attempt.